docs/technical-details/release-notes/4.tucker/README.md

```diff
@@ -17,9 +19,13 @@ HarperDB version 4 ([Tucker release](./tucker.md)) represents major step forward
 * Native caching capabilities for high-performance cache scenarios
 * Clone node functionality
 
-## [4.3](./4.3.0.md)
-* Relationships, joins, and broad new querying capabilities for complex and nested conditions, sorting, joining, and selecting with significant query optimizations
-* More advanced transaction support for CRDTs and storage of large integers (with BigInt)
-* Better management with new upgraded local studio and new CLI features
+## [4.1](./4.1.0.md)
+* New streaming iterators mechanism that allows query results to be delivered to clients _while_ querying results are being processed, for incredibly fast time-to-first-byte and concurrent processing/delivery
+* New thread-based concurrency model for more efficient resource usage
+
+## [4.0](./4.0.0.md)
+* New clustering technology that delivers robust, resilient and high-performance replication
```

Comment:
@jcohen-hdb  @kriszyp 

Introducing a new highly resilient and high-performance replication system designed to ensure robust data consistency and availability across distributed environments.

Major enhancements with a more efficient, adaptive-structure MessagePack format, featuring on-demand deserialization for faster data access and reduced overhead.